lib/matplotlib/backends/__init__.py

```diff
@@ -26,30 +29,31 @@ def pylab_setup(backend=None):
         The module which contains the backend of choice
 
     new_figure_manager : function
-        Create a new figure manage (roughly maps to GUI window)
+        Create a new figure manager (roughly maps to GUI window)
 
     draw_if_interactive : function
         Redraw the current figure if pyplot is interactive
 
     show : function
-        Show (and possible block) any unshown figures.
+        Show (and possibly block) any unshown figures.
 
     '''
     # Import the requested backend into a generic module object
-    if backend is None:
-        backend = matplotlib.get_backend()  # validates, to match all_backends
-
-    if backend.startswith('module://'):
-        backend_name = backend[9:]
+    if name is None:
```

Comment:
Is there any guarantee that `name`, if not None, is anything sensible that you would want to set to `backends.backend`?  For example, say I do `pylab_setup('keyboard')`. Obviously that's dumb since `keyboard` is not a valid mpl backend.  How should `pylab_setup` handle this?  I would imagine that you might want to do the `global backend; backend=name` at the very end right before the return?